Formula/mesa.rb

```diff
@@ -0,0 +1,368 @@
+class Mesa < Formula
+  include Language::Python::Virtualenv
+  desc "Graphics Library"
+  homepage "https://www.mesa3d.org/"
+  url "https://mesa.freedesktop.org/archive/mesa-18.2.3.tar.xz"
+  sha256 "e2bf83c17e1abdecb1ee81af22652e27e9aa38f963e95e60f34275cc0376304f"
+  depends_on "meson-internal" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python@2" => :build
+  depends_on "expat"
+  depends_on "freeglut" # for test
+  depends_on :x11
+
+  resource "Mako" do
+    url "https://files.pythonhosted.org/packages/eb/f3/67579bb486517c0d49547f9697e36582cd19dafb5df9e687ed8e22de57fa/Mako-1.0.7.tar.gz"
+    sha256 "4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
```

Comment:
Is mako used only for the build, or is it also needed after the install is done?

Reply:
According to [ArchLinux package](https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/mesa#n10)

Reply:
Then you need to install mako inside `buildpath`, not `libexec`.

Reply:
Done